### PR TITLE
Add configuration information to use MinIO for storage

### LINKED ISSUE #2720: Product Recipe: How to use Mattermost with Minio
Include setup, configuration, and talk about the Minio project

Category Suggestion: Storage

### PR DESCRIPTION
**Pull request by pravan**
I have added a relevant note along with every configuration option to configure Mattermost to use MinIO for storage using the Amazon S3 Driver.
I have created a local MinIO instance and configured my local Mattermost instance to use that MinIO instance and everything works perfectly.
Fix #2720.

### CODE REVIEW

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -2216,6 +2226,9 @@ Amazon S3 Access Key ID
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 This is required for access unless you are using an `Amazon S3 IAM Role <https://about.mattermost.com/default-iam-role-settings-documentation>`__ with Amazon S3. Your EC2 administrator can supply you with the access key ID.
 
+.. note::
+  For MinIO, you can find this in your terminal window with the name, ``AccessKey`` when you start a new MinIO server.
```
Comment (amyblais) [inline source/administration/config-settings.rst]:
```suggestion
  For MinIO, you can find this in your terminal window with the name ``AccessKey`` when you start a new MinIO server.
```

File: source/administration/config-settings.rst [hunk 2]
```diff
@@ -2226,6 +2239,9 @@ Amazon S3 Secret Access Key
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 The secret access key associated with your Amazon S3 Access Key ID.
 
+.. note::
+  For MinIO, you can find this in your terminal window with the name, ``SecretKey`` when you start a new MinIO server.
```
Comment (amyblais) [inline source/administration/config-settings.rst]:
```suggestion
  For MinIO, you can find this in your terminal window with the name ``SecretKey`` when you start a new MinIO server.
```

File: source/administration/config-settings.rst (hunk 1 above)
Comment (amyblais) [reply source/administration/config-settings.rst]: (Small tweak suggestion)

**Comment by pravan** [comment]
Thanks for the suggestions @amyblais ma'am. I have committed the suggestions.

**Comment by amyblais** [comment]
@pravan Merging, thank you again for the contribution!

**Pull request merged**